source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -36,13 +36,30 @@ How to migrate to Ember Octane 🐹🏗️,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Community Feedback 👏](https://www.outdoorsy.com)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Wondering about who has been using Ember Octane features in production? 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+We’ll spend some time over the next several issues focusing on projects and teams who are using Ember Octane in their apps.
+
+Today’s focus is on [**Outdoorsy**](https://www.outdoorsy.com), a peer-to-peer marketplace that connects RV and camper van owners with campers encouraging them to get into the great outdoors. Outdoorsy maintains three large-scale Ember apps for their flagship web product.
+
+The developers at Outdoorsy have been using Octane features in production since September and have reportedly had a super enjoyable experience. On the subject [Jaco Joubert @jacojoubert](https://github.com/jacojoubert), a Front-end Engineer at Outdoorsy, says "Octane has allowed us to build highly interactive experiences that are well tested in an unbelievably short time. Leveraging Octane we rebuilt our entire [search experience](https://www.outdoorsy.com/rv-search) from the ground up in less than five weeks, something which would have taken orders of magnitude longer before."
```

Comment:
The suggestion below removes "on the subject" and capitalizes "end" in "Front-End".
```suggestion
The developers at Outdoorsy have been using Octane features in production since September and have reportedly had a super enjoyable experience. [Jaco Joubert @jacojoubert](https://github.com/jacojoubert), a Front-End Engineer at Outdoorsy, says "Octane has allowed us to build highly interactive experiences that are well tested in an unbelievably short time. Leveraging Octane we rebuilt our entire [search experience](https://www.outdoorsy.com/rv-search) from the ground up in less than five weeks, something which would have taken orders of magnitude longer before."
```